<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CMakeOptionsDlg</class>
 <widget class="QDialog" name="CMakeOptionsDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>385</width>
    <height>118</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Make Graph Options</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QCheckBox" name="c_restrict_visibility">
       <property name="text">
        <string>Restrict visibile distance to </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="c_maxdist">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="c_boundarygraph">
     <property name="text">
      <string>Make boundary graph</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="c_ok">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="c_cancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>c_restrict_visibility</sender>
   <signal>clicked(bool)</signal>
   <receiver>CMakeOptionsDlg</receiver>
   <slot>OnRestrict(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>117</x>
     <y>28</y>
    </hint>
    <hint type="destinationlabel">
     <x>209</x>
     <y>46</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>c_ok</sender>
   <signal>clicked()</signal>
   <receiver>CMakeOptionsDlg</receiver>
   <slot>OnOK()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>104</x>
     <y>94</y>
    </hint>
    <hint type="destinationlabel">
     <x>29</x>
     <y>81</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>c_cancel</sender>
   <signal>clicked()</signal>
   <receiver>CMakeOptionsDlg</receiver>
   <slot>OnCancel()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>204</x>
     <y>96</y>
    </hint>
    <hint type="destinationlabel">
     <x>279</x>
     <y>82</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>OnRestrict(bool)</slot>
  <slot>OnOK()</slot>
  <slot>OnCancel()</slot>
 </slots>
</ui>
